lib/foreman_maintain/concerns/logger.rb

```diff
@@ -1,13 +0,0 @@
-require 'logger'
-
-module ForemanMaintain
-  module Concerns
-    module Logger
-      def logger
-        @logger ||= ::Logger.new($stderr).tap do |logger|
```

Comment:
could we still keep this module, just linking to the  `ForemanMaintain.logger` instance? Just for convenience of the code writers: less things to remember :)